models/generated/workloads/reservation.go

```diff
@@ -64,7 +64,7 @@ type ReservationData struct {
 	Networks                []Network             `bson:"networks" json:"networks"`
 	Kubernetes              []K8S                 `bson:"kubernetes" json:"kubernetes"`
 	Proxies                 []GatewayProxy        `bson:"proxies" json:"proxies"`
-	ReserveProxy            []GatewayReserveProxy `bson:"reserve_proxies" json:"reserve_proxies"`
+	ReverseProxy            []GatewayReverseProxy `bson:"reverse_proxies" json:"reverse_proxies"`
```

Comment:
Since we modify the name of the object in the database. we need a migration script that will rename this field. You can look at https://github.com/threefoldtech/tfexplorer/blob/master/migrations/container_missing_fields/main.go for some inspiration